pkgs/by-name/mq/mqttx-cli/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  mkYarnPackage,
+  fetchYarnDeps,
+  fetchFromGitHub,
+  lib,
+  testers,
+  mqttx-cli,
+}:
+
+mkYarnPackage rec {
```

Comment:
Could you use the yarn hooks instead?

#324246

Reply:
I was not aware of that hook. 
I'll take a look later.

Reply:
Fixed